core/src/main/kotlin/fr/sncf/osrd/api/api_v2/stdcm/STDCMEndpointV2.kt

```diff
@@ -91,9 +91,10 @@ class STDCMEndpointV2(private val infraManager: InfraManager) : Take {
                         workSchedules = request.workSchedules,
                         steps,
                         gridMarginBeforeTrain = request.timeGapBefore.seconds,
-                        gridMarginAfterTrain = request.timeGapAfter.seconds
+                        gridMarginAfterTrain = request.timeGapAfter.seconds,
+                        timeStep = request.timeStep!!.seconds
                     ),
-                    request.timeStep!!.seconds,
+                    request.timeStep.seconds,
```

Comment:
`request.timeStep` probably shouldn't be optional

Reply:
This is from the api, so I don't really wanna change that rn. Defaults to 2.0 as well. We'll circle back to it once you're back from vacation.